packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -96,8 +96,14 @@ async function createDataObject<
 	// access DDSes or other services of runtime as objects are not fully initialized.
 	// In order to use object, we need to go through full initialization by calling finishInitialization().
 	const scope: FluidObject<IFluidDependencySynthesizer> = context.scope;
-	const dependencyContainer = new DependencyContainer(scope.IFluidDependencySynthesizer);
-	const providers = dependencyContainer.synthesize<I["OptionalProviders"]>(optionalProviders, {});
+	const providers =
```

Comment:
We shouldn't take platform dependencies on scope, particularly since we want to remove it eventually.  Is there another way you can pass this through?

Reply:
We already are. This PR doesn't change that. I think it would be larger change and out of scope for this PR to remove providers from the PureDataObject constructor

Reply:
@ChumpChief you should open a bug here to get this fixed if you'd like it to work differently. I believe that work would fall under your team.